### ISSUE TITLE
Tabbing to clips in an audio track should give feedback for users of screen readers

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#1870]
The existing commands for selecting clips, moving to clip boundaries, etc provide feedback for users of screen readers.
When using tab or shift tab to move to the next/previous clip, there should be similar messages for users of screen readers.
